coralproject/talk
# Highlighting Links and New plugin: coral-plugin-comment-content

# Highlighting Links and New plugin: coral-plugin-comment-content

## What does this PR do?
This PR adds `coral-plugin-comment-content`.

- Enables full customization of the comment body/content.
- Enables highlighting for links in comments.
- Adds default components for Slots
- Replaces coral-ui path, by `plugin-api/beta/client/components/ui`
- Switch reaction plugins to components/ui
- Only create Slot elements once
- Support Class Components as defaultComponent

## How to test this PR?

- Add `coral-plugin-comment-content` to you plugins.json within `client`
- Post a comment with a link on it.
- Click it
- Check that everything is working as expected.

_Note: We will be migrating all the `coral-plugin-*` in the `client` folder to plugins._
